docs/01-start-here/06-faqs.md

```diff
@@ -0,0 +1,23 @@
+#FAQs
+
+## I can see mark-up in my response, but it's not in the application anywhere. Where is it coming from?
+
+Your markup is coming from CAPI and used as-is.
```

Comment:
I'd mention cleaners here - the CAPI response isn't necessarily being used as-is.